modules/vpn_ha/variables.tf

```diff
@@ -112,6 +112,7 @@ variable "tunnels" {
       advertise_groups    = optional(list(string))
       advertise_ip_ranges = optional(map(string))
       advertise_mode      = optional(string)
+      custom_learned_ip_ranges = optional(map(string))
```

Comment:
@angegar seems like it should be a list of strings not maps?

Reply:
@angegar according to API document you can have multiple blocks of custom_learned_ip_ranges. So it will be a good idea if you make it list of IP addresses and use dynamic block to loop through value and create multiple blocks of custom_learned_ip_ranges

https://cloud.google.com/compute/docs/reference/rest/v1/routers